src/stylesheets/components/_banner.scss

```diff
@@ -7,6 +7,10 @@
     font-size: font-size($theme-banner-font-family, "3xs");
     padding-bottom: units(0);
   }
+
+  .usa-accordion {
+    @include typeset($theme-banner-font-family);
```

Comment:
@thisisdano, this is under the assumption that `usa-accordion` font could be overridden right?

Reply:
yup — this can be set with the `$theme-accordion-font-family` theme variable — but in this instance we don't want that value to render, instead we want to render the value of `$theme-banner-font-family`

because those values happened to be the same in earlier testing, we didn't catch this